File: SmartValley.FrontEnd/src/app/api/project/my-project-response.ts
import {TeamMemberResponse} from './team-member-response';

export interface MyProjectResponse {
  id: number;
  name: string;
  externalId: string;
  category: number;
  stage: number;
  description: string;
  contactEmail: string;
  whitePaperLink: string;
  countryCode: string;
  website: string;
  icoDate: Date;
  facebook: string;
  bitcointalk: string;
  medium: string;
  reddit: string;
  telegram: string;
  twitter: string;
  github: string;
  linkedin: string;
  teamMembers: Array<TeamMemberResponse>;
}

File: SmartValley.WebApi/Projects/Responses/MyProjectResponse.cs
using System;
using System.Collections.Generic;
using SmartValley.WebApi.Projects.Requests;
using SmartValley.WebApi.WebApi;

namespace SmartValley.WebApi.Projects.Responses
{
    public class MyProjectResponse
    {
        public long Id { get; set; }

        public string Name { get; set; }

        public int Category { get; set; }

        public int Stage { get; set; }

        public string ImageUrl { get; set; }

        public string CountryCode { get; set; }

        public string Description { get; set; }

        public string Website { get; set; }

        public string WhitePaperLink { get; set; }

        public DateTimeOffset? IcoDate { get; set; }

        public string ContactEmail { get; set; }

        public string Facebook { get; set; }

        public string Linkedin { get; set; }

        public string BitcoinTalk { get; set; }

        public string Medium { get; set; }

        public string Reddit { get; set; }

        public string Telegram { get; set; }

        public string Twitter { get; set; }

        public string Github { get; set; }

        public IReadOnlyCollection<ProjectTeamMemberResponse> TeamMembers { get; set; }
    }
}